models/test/mapper/TestPreviewMapper.php

```diff
@@ -102,6 +102,7 @@ public function map(AssessmentTest $test, Route $route, TestPreviewConfig $confi
                     'flagged' => false,
                     'viewed' => false,
                     'categories' => $itemRef->getCategories()->getArrayCopy(),
+                    'allowSkipping' => $itemRef->getItemSessionControl()->doesAllowSkipping()
```

Comment:
This variable should be added to `\oat\taoQtiTestPreviewer\test\unit\models\test\factory\TestPreviewMapperTest`